en/docs/install-and-setup/setup/api-controller/managing-common-api-policies/migrating-common-api-policies-to-different-environments.md

```diff
@@ -99,3 +100,61 @@ The structure of an exported Common API Policy ZIP file is explained below:
         </tr>
     </tbody>
 </table>
+
+### Import a Common API Policy
+
+You can use the common API Policy archive exported from the previous section (or you can extract it and use the extracted folder) and import it to the WSO2 API-M instance in the target environment. 
+
+1.  Log in to the WSO2 API-M in the importing environment by following steps in [Login to an Environment]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller#login-to-an-environment).
+    
+    !!! tip
+        If you are already logged-in and your logged-in credentials and keys are already available in the `<USER_HOME>/.wso2apictl/keys.json` file, you can skip this step. 
+
+    !!! info
+        If you skip step 1 and if no keys exist for the environment in the `<USER_HOME>/.wso2apictl/keys.json` file, you will be prompt to log in to the environment when running the next command.
+
+2.  Run any of the following apictl commands to import a common API Policy.
+
+    -   **Command**
+        ``` bash
+        apictl import policy api -f <path to Common API Policy directory> -e <environment> 
+        ```
+        ``` bash
+        apictl import policy api --file <path to Common API Policy directory> --environment <environment>
+        ```
+        ``` bash
+        apictl import policy api -f  <path to Common API Policy archived file> -e <environment> 
+        ```
+        ``` bash
+        apictl import policy api --file <path to Common API Policy archived file> --environment <environment>
+        ```
+
+        !!! info
+            **Flags:**  
+            
+            -   Required :  
+                `--file` or `-f` : The file path of the common API Policy to import.  
+                `--environment` or `-e` : Environment to which the common API Policy should be imported.
+
+        !!! example
+            ```bash
+            apictl import policy api -f ~/addHeader_v1.zip -e production 
+            ```
+            ```bash
+            apictl import policy api --file ~/addHeader_v1.zip --environment production
+            ```   
+            ```bash
+            apictl import policy api -f ~/AddHeader -e production 
+            ``` 
+            ``` go
```

Comment:
Why do we have `bash` in all the other places and `go` in here? Any particular reason behind that?
```suggestion
            ``` bash
```